diff -up libdvdread-4.1.4/src/dvdread/dvd_reader.h.dvdstat libdvdread-4.1.4/src/dvdread/dvd_reader.h
--- libdvdread-4.1.4/src/dvdread/dvd_reader.h.dvdstat	2009-09-02 03:43:21.000000000 +0200
+++ libdvdread-4.1.4/src/dvdread/dvd_reader.h	2012-07-23 14:44:33.285613088 +0200
@@ -70,9 +70,13 @@ typedef struct dvd_reader_s dvd_reader_t
 typedef struct dvd_file_s dvd_file_t;
 
 /**
- * Opaque type that is used to provide statistics on a handle.
+ * Public type that is used to provide statistics on a handle.
  */
-typedef struct dvd_stat_s dvd_stat_t;
+typedef struct {
+  off_t size;          /**< Total size of file in bytes */
+  int nr_parts;        /**< Number of file parts */
+  off_t parts_size[9]; /**< Size of each part in bytes */
+} dvd_stat_t;
 
 /**
  * Opens a block device of a DVD-ROM file, or an image file, or a directory
diff -up libdvdread-4.1.4/src/dvd_reader.c.dvdstat libdvdread-4.1.4/src/dvd_reader.c
--- libdvdread-4.1.4/src/dvd_reader.c.dvdstat	2012-07-23 14:48:52.823635233 +0200
+++ libdvdread-4.1.4/src/dvd_reader.c	2012-07-23 14:50:09.689953096 +0200
@@ -111,12 +111,6 @@ struct dvd_file_s {
   ssize_t filesize;
 };
 
-struct dvd_stat_s {
-  off_t size;          /**< Total size of file in bytes */
-  int nr_parts;        /**< Number of file parts */
-  off_t parts_size[9]; /**< Size of each part in bytes */
-};
-
 int UDFReadBlocksRaw( dvd_reader_t *device, uint32_t lb_number,
                       size_t block_count, unsigned char *data,
                       int encrypted );
