diff -up wrk/src/ifo_read.c.wrk wrk/src/ifo_read.c
--- wrk/src/ifo_read.c.wrk	2016-05-18 13:53:53.979072295 +0200
+++ wrk/src/ifo_read.c	2016-05-18 13:54:24.760095439 +0200
@@ -1118,8 +1118,8 @@ void ifoFree_TT_SRPT(ifo_handle_t *ifofi
 
   if(ifofile->tt_srpt) {
     free(ifofile->tt_srpt->title);
-    free(ifofile->tt_srpt);
     ifofile->tt_srpt->title = NULL;
+    free(ifofile->tt_srpt);
     ifofile->tt_srpt = NULL;
   }
 }
